sample-lnd.conf

```diff
@@ -380,12 +380,9 @@ litecoin.node=ltcd
 ; watchtower.writetimeout=15s
 
 [wtclient]
-; Configure the private tower to which lnd will connect to backup encrypted
-; justice transactions. The format should be pubkey@host:port, where the port is
-; optional and assumed to be 9911 otherwise. At most one private tower URI is
-; supported at this time; if none are provided then the watchtower client will
-; be inactive.
-; wtclient.private-tower-uris=
+; Activate Watchtower Client. To get more information or configure watchtowers
+; run `lncli wtclient -h`
```

Comment:
nit: missing full stop at end of sentence.

Reply:
added